src/index.js

```diff
@@ -0,0 +1,18 @@
+/* eslint-disable import/no-named-as-default */
+import React from 'react';
+import ReactDOM from 'react-dom/client';
+import { BrowserRouter as Router } from 'react-router-dom';
+import { Provider } from 'react-redux';
+import App from './App';
+import './App.css';
+import store from './Redux/configStore';
```

Comment:
- [ ] Kindly fix the file reference for importing the store. It should be `./redux/configstore` with lowercase `R`.